configsuite/schema.py

```diff
@@ -49,7 +49,8 @@ def _check_required_type(schema_level):
 
 
 @configsuite.validator_msg(
-    "A type is not required only if it allows None or have a non-None default"
+    "A content can not be required and allow None, or unrequired and not allow None"\
```

Comment:
I agree that the original statement could benefit from a rewrite. I think the suggestion is wrong 😬 How about:

`A schema element can be required if and only if the schema element does not allow None as a value and defaults to None`?

Reply:
Is it wrong? I pretty much wrote out the logic in the return statement:
`return is_required != allow_none or has_non_none_default`

But I support your suggestion!